docs/docs/50-running/80-alpha-image-allow-rules.md

```diff
@@ -0,0 +1,154 @@
+# [Alpha Feature] ImageAllowRules (IARs)
+
+ImageAllowRules are an alpha-feature of Acorn, currently hidden behind a feature flag.
+To enable it in your Acorn installation, use `acorn isntall --features image-allow-rules=true` (**Beware**: first read the rest of this page before enabling it, as it's quite disruptive).
+
+## How ImageAllowRules work
+
+The principle behind IARs is to make your cluster more secure.
+In fact, secure by default, once it's enabled. No additional policies needed.
+If you enable this feature, you won't be able to deploy any Acorn image anymore without allowing it.
+To do that, you have to create an `ImageAllowRule` resource.
+If the app image is allowed by a single IAR in your project, it's good to run.
+
+**Note**: Removing an IAR won't stop your running app, but will update the `image-allowed` status condition on the app.
+
+## What makes up an ImageAllowRule
+
+Currently, IARs have two parts:
+
+1. The `images` scope (must-have), denoting, what images the rule applies to. It uses the same syntax as the auto-upgrade pattern. Examples below.
+2. The `signatures` rules (optional) define a set of image signatures and annotations on those signatures to make sure that an image was actually approved by someone or something, e.g. by your QA team. We're using [sigstore/cosign](https://docs.sigstore.dev/cosign/installation/) for everything related to signatures.
+
+## Example
+
+```yaml
+apiVersion: api.acorn.io/v1
+kind: ImageAllowRule
+metadata:
+  name: example-iar
+  namespace: acorn # your project namespace
```

Comment:
I think this is slightly clearer for any Acorn user's who don't know that project name == project namespace.

```suggestion
  namespace: acorn # your project name
```